src/TimeInput.spec.js

```diff
@@ -141,7 +141,6 @@ describe('<TimeInput />', () => {
       tree.find(Button).at(0).simulate('click')
 
       expect(getValue(tree)).toBe('13:37') // unchanged
-      expect(changeHandler).not.toHaveBeenCalled()
```

Comment:
Now is called in every date change and restored when canceled

Reply:
Why? There is no reason to call the change handler when the date isn't changed.

Reply:
I did that change because I needed the time text field (and some other components) to be updated when the user changes the hour in real time, so every time the user moves the clock hands, ``changeHandler`` is called.

Reply:
In fact, I think that's the best option, thus maintaining the same behavior (avoiding some developers from going crazy when updating 🙃) and providing that functionality, which in my case is very useful

Reply:
That was my intention. :+1: Adding your use case while keeping this a minor update (i.e. no breaking changes).